### ISSUE TITLE
The NLP NER example only predicts correct Os

### ISSUE THREAD

**Issue by rohun-tripathi** [cs230-stanford/cs230-code-examples#12]
Hi, the NER example has an accuracy of 0.82 or 0.78 after running for 10 epochs on the toy 10 values dataset. I was using this code to startup a NER task I have to do. It is just predicting 'O's. I checked on set of predictions at the end of the epochs and the I's and B's predicted were all wrong.

Use a score like F1 on the B/I to get a better idea.

Note - 
I am not a student in the course. and I know it is supposed to be starter code, but the way it is presented, one expects that it'll work for the toy example.

**Comment by suragnair** [comment]
Is this the tensorflow or the pytorch version?

**Comment by rohun-tripathi** [comment]
The PyTorch version
